data/pigui/modules/flight-ui/reticule.lua

```diff
@@ -33,6 +41,11 @@ local player = nil
 -- this should go into HUD settings
 local showNavigationalNumbers = true
 
+-- to interact with actions and axes in the input system
+local bindings = {}
+bindings.assistRadial = bindManager.registerAction('BindFlightAssistRadial')
+bindings.fixheadingRadial = bindManager.registerAction('BindFixheadingRadial')
```

Comment:
Seems a little strange to create the table and then assign into it when you don't need to back-reference the table; I'd prefer this form:

```suggestion
local bindings = {
    assistRadial = bindManager.registerAction('BindFlightAssistRadial'),
    fixheadingRadial = bindManager.registerAction('BindFixheadingRadial')
}
```

Reply:
Done!